### DISCUSSION TITLE
Many to Many Filter only object Output?

### DISCUSSION THREAD

**Discussion by Bastilla123** [carltongibson/django-filter#1422] [Q&A]
i have installed per pip django-filters.

Models.py:

class Productinterests(models.Model):
    id = models.AutoField(primary_key=True)
    name = models.CharField(max_length=255, default="", blank=False, null=False)

class Address(models.Model):
   productinterests = models.ManyToManyField(Productinterests,blank=True, default=0)
   lastname = models.CharField(max_length=255, default="", )
   firstname = models.CharField(max_length=255, default="", blank=True, null=True)

filters.py:
   class AddressFilter(django_filters.FilterSet):
lastname = django_filters.CharFilter(lookup_expr='icontains',label="Nachname")
firstname = django_filters.CharFilter(lookup_expr='icontains', label="Vorname")

    productinterests = django_filters.ModelMultipleChoiceFilter(to_field_name='name',queryset=Productinterests.objects.all(),
widget=forms.CheckboxSelectMultiple, label="Produktinteresse")

    class Meta:
       model = Address
       fields = ['lastname','firstname','productinterests' ]

views.py:
   def search(request):
         liste = Address.objects.all()

         address = AddressFilter(request.GET, queryset=liste)
return render(request, 'search/address_list.html', {'filter': user_filter})

**Problem: Website shows in Multiselect Productinterest only as Output:
 "Productinterests object (1)" instead of name from object. How can i output name?**

**Comment by carltongibson** [comment]
Implement `__str__` on your model. See https://docs.djangoproject.com/en/3.2/ref/models/instances/#str
